### DISCUSSION TITLE
backups naar synology?

### DISCUSSION THREAD

**Discussion by gobartos** [dsmrreader/dsmr-reader#1484] [Questions & answers / How-to's] [answered]
Ik heb DSMR-reader op dezelfde VM als Home Assistant geïnstalleerd, met Proxmox als host.
Zoals je merkt ken ik weinig van Linux, dus mijn excuses. Maar op welke manier kan ik de backup naar mijn Synology maken? Die beschikt over Linux indeling indien gewenst.

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag. Ik kan je hier helaas niet mee helpen en alleen doorverwijzen. Je kunt de link in #701 proberen en anders ben je denk ik het het beste geholpen op een Synology-forum, HA-forum of het Tweakers-forum. 

In de DSMR-reader admin kun je iig een map opgeven waar de backup's heen gaan. Standaard is dat `backups/`, gezien vanaf `/home/dsmr/dsmr-reader/`, dus absoluut pad is `/home/dsmr/dsmr-reader/backups/`. 
Je kunt ook (indien nodig) een ander, absoluut pad opgeven, mits de `dsmr` user op je VM naar dat pad kan schrijven. 

Het uitzoekwerk zit hem dan nog voor je in om bovenstaande map periodiek naar je NAS te krijgen.

**Comment by gobartos** [comment]
Bedankt, dat is alvast een duwtje in de goeie richting. Ik vrees dat er toch erg veel data op mij af komt :)

**Comment by dhover** [comment]
Sinds de mogelijkheid van het toevoegen van extra mounts in Home Assistant is het relatief eenvoudig om de backup bestanden van DSMR Reader naar de Synology te redirecten.
In de Synology maak je eerst een share aan t.b.v. deze backups. Vervolgens maak je een user aan met schrijfrechten op deze share. In dit voorbeeld heb ik een share "backup" op de Synology gemaakt.
In Home Assistant maak je nu een mount aan naar deze share. Dit gaat als volgt:
System, Storage, Add Network Storage.
Name: bijv. DsmrReaderBackup
Usage: Share !!!  (dit is belangrijk want backup werkt alleen voor HA backups)
Server: ip adres van de synology
Protocol: Samba/Windows (CIFS)
Remote share: backup/dsmrreader Hierbij is "backup' de naam van de share op de Synology en "dsmrreader" een map binnen deze share
Username: username van de Synology gebruiker
Password: wachtwoord van de Synology gebruiker
Nu als laatste in DSMR Reader bij Backup-upconfiguratie invullen /share/DsmrReaderBackup
Let op de hoofd- en kleine letters.
/share verwijst naar de usage van het type Share en DsmrReaderBackup is de naam van de mount
In de DSMR Reader kun je op het knopje Nu drukken om een backup te forceren.
